### DISCUSSION TITLE
Is vercel app down?

### DISCUSSION THREAD

**Discussion by Rounak-stha** [vercel/community#1046] [Help] [answered]
When trying to import a github repo on vercel I am getting a timeout error by github and when trying to access the vercel app on github, I get the same error. What seems to be the issue?

**Comment by swarnava** [comment, marked as answer]
Hi, that's a known bug from GitHub side. You need to leave this organization: https://github.com/me50 to fix it.

**Reply by Rounak-stha** [reply]
Thank you for the answer; the issue is fixed.
